awx/api/serializers.py

```diff
@@ -3061,6 +3070,11 @@ def get_related(self, obj):
                 res['project_update'] = self.reverse('api:project_update_detail', kwargs={'pk': obj.project_update.pk})
         except ObjectDoesNotExist:
             pass
+        try:
+            if obj.instance_groups:
+                res['instance_groups'] = self.reverse('api:job_instance_group_list', kwargs={'pk': obj.pk})
+        except ObjectDoesNotExist:
+            pass
```

Comment:
I don't think this is the right way to use `ObjectDoesNotExist`, because it's a relationship, and `obj.instance_groups` should always exist, so better to get rid of the try-except.

Reply:
I think initially I was thinking incase an IG got deleted but since they are FK constraints that wouldn't matter, updated.